cli/installer/install-azd.ps1

```diff
@@ -55,17 +56,53 @@ param(
     [string] $BaseUrl = "https://azure-dev.azureedge.net/azd/standalone/release",
     [string] $Version = "latest",
     [switch] $DryRun,
-    [string] $InstallFolder = "$($env:LocalAppData)\Programs\Azure Dev CLI",
+    [string] $InstallFolder,
     [switch] $NoPath,
     [int] $DownloadTimeoutSeconds = 120
 )
 
-if ($IsLinux -or $IsMacOS) {
-    Write-Error "This bootstrap script is intended to run on Windows. Use the install-azd.sh instead."
-    exit 1
+function isLinuxOrMac {
+    return $IsLinux -or $IsMacOS
+}
+
+if (!$InstallFolder) {
+    $InstallFolder = "$($env:LocalAppData)\Programs\Azure Dev CLI"
+    if (isLinuxOrMac) {
+        $InstallFolder = "/usr/local/bin"
```

Comment:
Nothing blocking but longer term I wonder if this should be in some part of the user's HomeDir, as to not require `sudo`.  If we do that we probably want to read https://wiki.archlinux.org/title/XDG_Base_Directory and see what location we should default to (I would guess `XDG_DATA_HOME`).

Reply:
https://github.com/Azure/azure-dev/issues/219